### PR TITLE
add more similarity

### PR DESCRIPTION
**Pull request by GabrielBlondeau-Kirby**
Just add like this:

> Definition

DELETE https://api.shareactor.io/issuers/<issuer_id>

> Example request:

### CODE REVIEW

File: source/includes/_issuers.md [hunk 1]
```diff
@@ -206,6 +224,12 @@ modified | Date generated when Issuer is updated.
 
 ## Update Issuer
 
+> Definition
+
+PUT https://api.shareactor.io/issuers/issuer_id
```
Comment (pedropregueiro) [inline source/includes/_issuers.md]: use `<issuer-id>` instead:
```
PUT https://api.shareactor.io/issuers/<issuer-id>
```

**Comment by GabrielBlondeau-Kirby** [comment]
ready for new review :)
@pedropregueiro  @KristopherJordan

File: source/includes/_issuers.md [hunk 2]
```diff
@@ -70,25 +76,31 @@ Content-Type: application/json
...
-address | Issuer address. As an [`Address`](#address) object.
-is_messaging_mandatory | Determines if *message* field is mandatory for invoices issued by this issuer. `boolean`. Default value `false`.
+Arguments | Type | Description
+--------- | ---- | ------
+**created** | `timestamp` | Created is date generated when Issuer created. _default value `now date`_
+**modified** | `timestamp` | Date generated when Issuer is updated. _default value `now date`_
+**active** | `boolean` | Flag that sets Issuer object to active. _default `true`_
+**deleted** | `boolean` | Flag that sets Issuer object to deleted. _default `false`_
+**verified** | `boolean` | Flag that sets Issuer object to verified. _default `false`_
+**blacklisted** | `boolean` | Flag that sets Issuer object to blacklisted. _default `false`_
```
Comment (KristopherJordan) [inline source/includes/_issuers.md]: Are you sure we should have these fields? From created to blacklisted. These are automatically set in the API. I think it's better to explain fields like these in the method object introduction.

File: source/includes/_issuers.md [hunk 3]
```diff
@@ -306,30 +336,33 @@ Content-type: application/json
...
 Retrieves a list of Issuers associate with search.
 
-Arguments | Description
+Arguments | Type | Description
 --------- | -----------
-**query** | What you want to search, like **name** or **account_number**. `string`
-filters | In search you can use filters for more accurate search.
-
+**query** | `string` | What you want to search, like **name** or **account_number**.
+filters | `string` | In search you can use filters for more accurate search.
```
Comment (KristopherJordan) [inline source/includes/_issuers.md]: Use filters for more accurate results.

File: source/includes/_issuers.md (hunk 2 above)
Comment (GabrielBlondeau-Kirby) [inline source/includes/_issuers.md]: >Are you sure we should have these fields? 

Yes I'm

> These are automatically set in the API

Hmmm Are you sure ?

> I think it's better to explain fields like these in the method object introduction.

I agree

**Pull request merged**